src/Compilers/CSharp/csc/project.json

```diff
@@ -3,11 +3,11 @@
     "Microsoft.DiaSymReader.Native": "1.5.0-beta1"
   },
   "frameworks": {
-    "net46": { }
+    "net46": {}
   },
   "runtimes": {
-    "win7": { },
-    "ubuntu.14.04": { },
-    "osx.10.10": { }
+    "win7": {},
+    "ubuntu.14.04": {},
+    "osx.10.10": {}
```

Comment:
>osx.10.10": {} [](start = 5, length = 14)

Why do we have non-Windows entries here?  This is the desktop CSC project hence only runs on full framework.  The CscCore is our cross plat entry point.